tests/v2/validation/auth/userretention/userretention.go

```diff
@@ -0,0 +1,166 @@
+package userretention
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/rancher/machine/libmachine/log"
```

Comment:
is there a reason why we are using this instead of "github.com/sirupsen/logrus"?